source/develop/release-management/features/storage/incremental-backup.html.md

```diff
@@ -250,6 +253,16 @@ socket:
     "url": "nbd:unix:/run/vdsm/nbd/xxxyyy-nbd.sock:exportname=/sda"
 }
 ```
+
+### Open Issues
+
+- What should we do when VM is stopped? Cancel backup flow?
+
+- How to handle entities locks upon backup?
```

Comment:
Backup should lock disks, we cannot depend on locking during transfer. If we lock in backup, we need a way to transfer the locked disks.